src/quo2/components/drawers/documentation_drawers/style.cljs

```diff
@@ -1,9 +1,14 @@
-(ns quo2.components.drawers.documentation-drawers.style)
+(ns quo2.components.drawers.documentation-drawers.style
+  (:require [react-native.safe-area :as safe-area]))
+
+(def outer-container
+  ;; Margin negative to fill the bottom-sheet container where this component is used
+  {:margin-bottom (- (+ (safe-area/get-bottom) 8))})
```

Comment:
The bottom sheet component adds a padding at the bottom of the component to not draw content over the iOS swipe bar, but the documentation drawer needs to be drawn over the swipe bar when the content is long, so this negative margin has been added

Reply:
I'm not sure, but I felt the `bottom-sheet` component should give us the ability to toggle bottom padding as per the content needs. It helps us to prevent these workarounds in the content.

Reply:
the `bottom-sheet` let us override its padding bottom using the property `padding-bottom-override`, but that means that every time we show a bottom sheet that is showing a `documentation-drawer` we call it like this:

```clojure
(rf/dispatch [:show-bottom-sheet
              {:content                 connectivity-sheet
               :padding-bottom-override 0}])
```

which is an error-prone approach since the developer needs to remember adding `:padding-bottom-override 0` to the dispatch event every time a documentation drawer is shown to match figma designs.

I know that workaround (negative margin) doesn't  look good, but I prefer it.

Instead, I think it'd be better that the documentation drawer never adds that padding at the bottom, since the appearence of the content is responsibility of the different drawers (action drawer, doc drawer, etc). but it is not done in this way.